ecmascript/ast/src/lib.rs

```diff
@@ -142,3 +142,72 @@ impl Default for TargetEnv {
         TargetEnv::Browser
     }
 }
+
+/// List of built in packages for latest stable node with LTS (node@16).
+///
+/// Run `node -p "require('module').builtinModules"`
+pub const NODE_BUILTINS: [&'static str; 63] = [
```

Comment:
I think it is not a good place for this varianble.
Can you move this to `swc_ecma_loader/node`?

Reply:
Also, I think `&[&str]` is better because it reduces breaking changes.

Reply:
Cool, I put it at `swc_ecma_loader::NODE_BUILTINS` because `swc_ecma_loader::resolvers::node` is behind a feature flag and this should be accessible without the flag IMO. I wonder whether putting `NodeModulesResolver` behind a feature flag is strictly necessary?

Reply:
> I wonder whether putting NodeModulesResolver behind a feature flag is strictly necessary?

Yes, but it's not necessary for `NODE_BUILTINS`.

I want to avoid regressing deno, but single const will be easily stripped out by the compiler, without regressing compile time by a margin.